Node-1st-gen/quickstarts/uppercase-firestore/functions/index.js

```diff
@@ -18,7 +18,7 @@
 // [START all]
 // [START import]
 // The Cloud Functions for Firebase SDK to create Cloud Functions and set up triggers.
-const functions = require("firebase-functions");
+const functions = require('firebase-functions/v1');
```

Comment:
FYI @egilmorez this change touches the 1st gen "get started" snippet: https://firebase.google.com/docs/functions/get-started?gen=1st#import-the-required-modules-and-initialize-an-app_1

Reply:
Yup!  As long as the tag names dont' change, I'll just need to republish to catch this and any other updates to snippets.